frontend/src/stores/useProjectStore.ts

```diff
@@ -212,8 +219,8 @@ const useProjectStore = create<ProjectStore>()(persist((set: SetState<ProjectSto
   })),
 
   /* Create a new comment */
-  createComment: (comment: ProjectComment) => {
```

Comment:
I updated the type in the interface and with type inference we don't need the type here also

Reply:
Hmm, I noticed that doing the same thing for `createState` (removing explicit type of `AutomataState`) results in an error if the inferred type of `Omit<AutomataState, 'isFinal' | 'id'>` is used, seeing as the `isFinal` attribute is optional as opposed to omitted and it used in the function body. Perhaps this typing should be changed for consistency? The typing for all of the comment stuff is good though

Reply:
Will look into it 👍